velox/expression/Expr.h

```diff
@@ -373,6 +379,10 @@ class Expr {
     return stats_;
   }
 
+  ExprStats& mutableStats() {
```

Comment:
is there any use case where we might want this? I wonder if we should just have an API in ConstantExpr class that would update the stat `defaultNullRowsSkipped`, and mention what its used for (only during constant folding).

Also, do we plan to remove all the logic around `defaultNullRowsSkipped` once presto and velox are in sync on this behavior?

Reply:
since you have switched to using `setDefaultNullRowsSkipped`, should we get rid of this?